python/paddle/fluid/transpiler/collective.py

```diff
@@ -173,8 +185,8 @@ class GradAllReduce(Collective):
     '''
     '''
 
-    def __init__(self):
-        Collective.__init__(self)
+    def __init__(self, nrings=2):
```

Comment:
why 2?

Reply:
The minimum number for parallel comms/streams. As it is no harm for paralleling collective communication in GradAllReduce mode, we prefer this value to 1 which refers to no parallel at all.